2017/ru/0x00-notice.md

```diff
@@ -0,0 +1,22 @@
+# Релиз
+
+## Важное сообщение
+
+### Запрос комментариев
+
+Эта текстовая версия Топ-10 OWASP будет полезна переводчикам и другим заинтересованным лицам. Данная версия не является официальным релизом, который будет представлен документами в формате PowerPoint и PDF.
```

Comment:
Please limit the line length with 80 or 120 chars (markdown supports soft breaks): it's more convenient for diff. Also, see [my explanation](https://gist.github.com/KvanTTT/acbf3c63cec69bcee23bae8f2f5f69ee) :)

Reply:
I'll try that. I think it's quite reasonable.